### ISSUE TITLE
more support for cortana

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5218]
**Reported by mohammadobid on 2015-07-13 10:18**
I can't get to my reminders and my notebook in cortana.
It is not accessible with nvda.
Can you improve its accessibility?

**Comment by ehollig** [comment]
I am able to create and read reminders with cortana in Notebook using Windows 10,1607. NVDA Master 14175

**Comment by leonardder** [comment]
Sounds like a works for you, so it will probably work for me as well. Closing, but please ring a bell when there's still something more or less horribly wrong.

**Comment by josephsl** [comment]
NO, not just yet.

**Comment by josephsl** [comment]
There are more work to be done.

Technical: issue caused by EdgeHTML code. Back when I (nvdakor) commented on this, NVDA did not have proper Edge support. Now that it does, it can be closed now. But there is an important thing to be done, and that'
s a separate issue for which I wrote a solution.

Thanks.
